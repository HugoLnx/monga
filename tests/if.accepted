int main() {
  if(x > 10) {
    print("hey! ho! lets go!");
  }
}
-----
int main() {
  if(x > 10) {
    print("hey! ho! lets go!");
  }
  else{
  	printf();
  }
}
-----
int main() {
  if(x > 10) {}
  else{
  	printf();
  }
}
-----
int main() {
	if(10.8){

	}
}