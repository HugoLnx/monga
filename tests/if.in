int main() {
  if(x > 10) {
    a = 2;
    
    if(x > 10) {
    	print("hey! ho! lets go!");
  	}
  	else
  		i = 2 > 5;
  }
  
  else{
  	if(x > 10) {
    	y = 2 / 4;
  	}
 	
 	while(10.8){

 	}
  }
}