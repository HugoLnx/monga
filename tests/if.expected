var TK_ONE_EQUAL exp
attribution TK_SEMICOLON
function_call TK_SEMICOLON
block
TK_IF TK_PARENTHESES_OPEN exp TK_PARENTHESES_CLOSE statement
attribution TK_SEMICOLON
if TK_ELSE statement
TK_CURLY_BRACE_OPEN statement_list TK_CURLY_BRACE_CLOSE
block
TK_IF TK_PARENTHESES_OPEN exp TK_PARENTHESES_CLOSE statement
block
TK_IF TK_PARENTHESES_OPEN exp TK_PARENTHESES_CLOSE statement
if 
TK_WHILE TK_PARENTHESES_OPEN exp TK_PARENTHESES_CLOSE statement
block
if TK_ELSE statement
